runtime/onert/backend/gpu_cl/open_cl/CL/opencl.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2021 Samsung Electronics Co., Ltd. All Rights Reserved
+ * Copyright (c) 2008-2020 The Khronos Group Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __OPENCL_H
+#define __OPENCL_H
+
```

Comment:
About overall open_cl headers put on here are partially merged with a slight modification.
how about keep headers as same as original codes from tensorflow's or opencl reference.
Removed codes because of unused warning errors can avoid errors with # pragma options
There's an example on arm_compute/core/CL/OpenCL.h. if possible I suggest keep those files as externals.

Reply:
yes. I will change to download open cl header file before build. I'll add it then.